src/passes/OptimizeInstructions.cpp

```diff
@@ -1422,11 +1441,9 @@ struct OptimizeInstructions
       return curr;
     }
     // (unsigned)x / -1   ==>   x == -1
-    // TODO: i64 as well if sign-extension is enabled
-    if (matches(curr, binary(DivUInt32, any(), ival(-1)))) {
-      curr->op = Abstract::getBinary(type, Abstract::Eq);
-      return curr;
-    }
+    // We handled earlier as part of (unsigned)x / C rule
+    // where `C > (2^31|2^63)`
```

Comment:
This whole comment (including the part that was already there) can probably just be removed.